src/main/java/org/wso2/carbon/connector/utils/Utils.java

```diff
@@ -0,0 +1,112 @@
+/*
+ * Copyright (c) 2021, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.wso2.carbon.connector.utils;
+
+import org.apache.avro.AvroTypeException;
+import org.apache.avro.SchemaBuilderException;
+import org.apache.avro.SchemaParseException;
+import org.apache.avro.UnresolvedUnionException;
+import org.apache.axis2.AxisFault;
+import org.apache.kafka.common.errors.SerializationException;
+import org.apache.synapse.MessageContext;
+import org.apache.synapse.SynapseConstants;
+import org.wso2.carbon.connector.core.ConnectException;
+import org.wso2.carbon.connector.exception.InvalidConfigurationException;
+
+import java.io.PrintWriter;
+import java.io.StringWriter;
+import java.io.Writer;
+
+public class Utils {
+
+    /**
+     * Sets error to the message context
+     *
+     * @param msgCtx Message Context to set info
+     * @param e      Exception associated
+     * @param error  Error code
+     */
+    public static void setError(MessageContext msgCtx, Exception e, Error error) {
+
+        setErrorPropertiesToMessage(msgCtx, e, error);
+    }
+
+    /**
+     * Sets the error code and error detail in message
+     *
+     * @param messageContext Message Context
+     * @param error          Error to be set
+     */
```

Comment:
Lets update the java doc for exception parameter

Reply:
Fixed with ba1167b